.github/workflows/ci.yml

```diff
@@ -51,27 +51,13 @@ jobs:
     strategy:
       matrix:
         include:
-            - os: ubuntu-20.04
-              python-version: 3.6
-              toxenv: py36-fuse2
-            - os: ubuntu-20.04
+            - os: windows-latest
               python-version: 3.7
-              toxenv: py37-fuse2
-            - os: ubuntu-20.04
-              python-version: 3.8
-              toxenv: py38-fuse3
-            - os: ubuntu-20.04
-              python-version: 3.9
-              toxenv: py39-fuse3
-            - os: macos-latest
-              # note: it seems that 3.8 and 3.9 are currently broken,
-              # neverending RuntimeError crashes...
-              python-version: 3.7
-              toxenv: py37-fuse2
+              toxenv: py37-none
```

Comment:
this is only temporary, so that tweaking the windows CI does not cause a lot of unnecessary load by the linux/macOS builds. before merging this PR, this commit needs to be removed: d615df0aedd18dfaedf347736c000f18348c5541